roles/podman/templates/container-.service.j2

```diff
@@ -22,7 +22,6 @@ ExecStartPre=/bin/rm -f %t/%n-pid %t/%n-cid
 # to the host adapter. Had issues with ufw conflicting when
 # using --publish directly. See: https://stackoverflow.com/a/51741599
 ExecStart=/usr/bin/podman run \
-  --cgroups=no-conmon \
```

Comment:
We should probably use the no-conmon option rather than completely disable cgroups. As it is the default option when generating a new podman service file with --new. Maybe @jchristgit remembers why there is a disabled on the cgroup here?

https://docs.podman.io/en/latest/markdown/podman-generate-systemd.1.html

Reply:
In #18, one of the goals is

> Configure Podman services to use host cgroup and run managed by systemd

According to @jchristgit, we want to use systemd for resource limits (meaning we don't need pdoman to create cgroups) for "ease of getting metrics."

Reply:
Oh yeah, that makes sense. 😄